pkg/server/cluster_definition_handler.go

```diff
@@ -0,0 +1,95 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/url"
+)
+
+const authTokenRequestURL = "https://developers.redhat.com/auth/realms/rhd/protocol/openid-connect/token"
+const pullSecretRequestURL = "https://api.openshift.com/api/accounts_mgmt/v1/access_token"
+
+type ClusterDefinition struct {
+	ClusterName string `json:"clusterName"`
```

Comment:
Snake case might be better than camel case.

Reply:
Done, although it requires pre-processing the data on the JS side. Unless there is a possibility to set json flags same as we do in Go, which I don't know about.